backend/app/views/backend/shared/_index_items_search_and_filter.html.erb

```diff
@@ -0,0 +1,16 @@
+<div class="btn js-index-searcher-button">
+  <svg class="icon">
+    <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-search"></use>
+  </svg>
+</div>
+<div class="btn">
+  <svg class="icon">
+    <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-sort"></use>
+  </svg>
+</div>
+<div class="searcher js-index-searcher" <% if @search.present? %>style="display: block;"<% end %>>
```

Comment:
If you push the 'esc' key, the searcher will be disappeared and the default items will be reloaded... what do you think? is a heavy user behaviour?